src/app/utils/convert-html-to-pdf.ts

```diff
@@ -0,0 +1,33 @@
+import puppeteer from 'puppeteer-core'
+
+import config from '../config/config'
+
+/**
+ * Utility function to generate a PDF from HTML.
+ * Used to send autoreply emails, and to generate payment receipts
+ * @param summaryHtml HTML to generate PDF from
+ * @returns PDF buffer
+ */
+export const generatePdfFromHtml = async (
+  summaryHtml: string,
+): Promise<Buffer> => {
+  const browser = await puppeteer.launch({
```

Comment:
Let's wrap this function with a DD span, so we can monitor how long the pdf generation takes.

Reply:
[a1e656d](https://github.com/opengovsg/FormSG/pull/5837/commits/a1e656d1de54860d037c5e4591a7c0470c22fff9)